connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectAddArtifactsHandler.scala

```diff
@@ -128,7 +136,7 @@ class SparkConnectAddArtifactsHandler(val responseObserver: StreamObserver[AddAr
       responseObserver.onNext(builder.build())
       responseObserver.onCompleted()
     } catch {
-      case e: StatusRuntimeException => onError(e)
+      ErrorUtils.handleNonQueryError("addArtifact - onNext", responseObserver, Some(this.holder))
```

Comment:
nit: should this op name be "addArtifact - onCompleted"?
nit2: the RPC is named "addArtifacts"; should we use opname "addArtifacts"?
nit3: maybe "addArtifacts.onNext" / "addArtifacts.onCompleted"? I somehow find the " - " separator weird.

Reply:
You could have saved yourself a large number of characters by just proposing the last one. Changing.